elastic/logs/challenges/categorize-text.json

```diff
@@ -1,5 +1,5 @@
 {% import "rally.helpers" as rally %}
-{% set shard_size_list = [10, 100, 1000] %}
+{% set shard_size_list = [10, 100, 1000, 1000000000] %}
```

Comment:
This "unlimited" shard_size is (currently) necessary to fairly compare ES|QL and query DSL, because ES|QL only supports unlimited shard_size.